### DISCUSSION TITLE
Issue with Breeze theme

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#227] [1. Help] [answered]
Hi @vovayatsyuk 

We recently installed a breeze theme in our cloud environment, and it's working fine locally but not on integration.

If I compare body attributes, some are missing on Integration

from local
`<body data-container="body" data-mage-init="{&quot;loaderAjax&quot;: {}, &quot;loader&quot;: { &quot;icon&quot;: &quot;http://kewcloud.local.com/static/version1704698943/frontend/Swissup/breeze-blank/en_US/images/loader-2.gif&quot;}}" class="breeze breeze-theme cms-home page-layout-1column cms-index-index page-layout-cms-full-width" id="html-body" data-breeze-processed="true" data-new-gr-c-s-check-loaded="14.1148.0" data-gr-ext-installed="" cz-shortcut-listen="true">`

from Integration
`<body data-new-gr-c-s-check-loaded="14.1148.0" data-gr-ext-installed="" cz-shortcut-listen="true">`

We have purchased themes and subscriptions and added domains as well.
Can you please suggest anything on this?

Thanks,
Pankaj

**Comment by vovayatsyuk** [comment]
I've seen similar issues in the past. A broken markup in the head caused them. Check your head section for invalid markup also (images, divs, and other prohibited tags)

**Reply by pk-pankaj** [reply]
I tried but could not find anything suspicious can you please review our integration URL https://integration-5ojmyuq-krhp5ka5aubqw.eu-3.magentosite.cloud/ ?

**Reply by vovayatsyuk** [reply]
noscript is prohibited in head:

![image](https://github.com/breezefront/community/assets/306080/a64f6953-9a72-4734-bdc4-cdca04d55780)

**Reply by pk-pankaj** [reply]
I have removed that one but the issue is still the same, could you please review it again?

**Reply by vovayatsyuk** [reply]
I don't see anything either. Maybe it was removed by the PHP dom parser? Can you check misc field in admin?

Can you also enable breeze debug mode, so breeze=0 will disable breeze?

**Reply by pk-pankaj** [reply]
@vovayatsyuk There is nothing in misc field, I have enabled debug mode, can you please check now if get anything https://integration-5ojmyuq-krhp5ka5aubqw.eu-3.magentosite.cloud/, please remember it's working fine on local environment.

**Reply by vovayatsyuk** [reply, marked as answer]
Please look at https://integration-5ojmyuq-krhp5ka5aubqw.eu-3.magentosite.cloud/?breeze=0

There are multiple errors in head:

```
<noscript>
    <img height="1" width="1" style="display:none"
     src="https://www.facebook.com/tr?id=328694677943998&ev=PageView&noscript=1&a=magento2-0.0.0-1.4.6" />
        </noscript>

...

<div id="newsmodal-popup" class="modal newsletter-popup-wrapper" style="display:none;">
        <div class="modal-content">           
            <div class="content_area">
....
```

**Reply by pk-pankaj** [reply]
It worked thanks @vovayatsyuk
